rhttpc-client/src/main/scala/rhttpc/client/proxy/ReliableProxy.scala

```diff
@@ -38,9 +38,10 @@ class ReliableProxy[Req, Resp](subscriberForConsumer: ActorRef => Subscriber[Cor
                                handleResponse: Correlated[Exchange[Req, Resp]] => Future[Unit],
                                additionalStartAction: => Unit,
                                additionalStopAction: => Future[Unit],
-                               timeProvider: => Instant)
+                               firstAttemptTimeProvider: => Instant,
```

Comment:
Why you've added new time provider? `nowProvider` is enough...

Reply:
I think that I know the purpose. Let's it stay as it is...

Reply:
I thought that since we have 2 timestamps that we might want to test / set separately both date of first attempt of a request and what time is "now" for the sake of testing behaviour of this new strategy